Documentation/Blazorise.Docs/Models/Snippets.generated.cs

```diff
@@ -2885,27 +2885,78 @@ void ValidateCheck( ValidatorEventArgs e )
 
         public const string VideoScriptsExample = @"<script src=""_content/Blazorise.Video/video.js"" type=""module""></script>";
 
-        public const string AnimateExample = @"<Animate @ref=""animateRef"" Auto=""false"" Animation=""Animations.FadeIn"" DelayMilliseconds=""500"">
-    <Card Margin=""Margin.Is4.OnY"">
-        <CardBody>
-            <CardTitle Size=""5"">Fade Example</CardTitle>
-            <CardText>
-                Some content.
-            </CardText>
-        </CardBody>
-    </Card>
-</Animate>
-<Button Color=""Color.Primary"" Clicked=""@Animate"">Animate!</Button>
+        public const string AnimateExample = @"<Select TValue=""IAnimation"" @bind-SelectedValue=""selectedAnimation"">
+
+    <SelectItem Value=""Animations.Fade"">Fade</SelectItem>
```

Comment:
I think it would be better to loop the enum Animations values instead of defining them manually.

Reply:
It's not an enum... it's a static class.. couldn't be bothered with figuring out a way to do reflection....
If you got a code suggestion ready... I'll take it tough.... haha

Reply:
nvm, I stopped being lazy and just did it...